docs/data_management/transfer/index.md

```diff
@@ -9,3 +9,41 @@ We currently support the following tools on Cheaha to manage data transfers. You
 We also have tutorials to guide your data transfer needs;
 
 1. [Data Transfer (Globus)](../transfer/tutorial/index.md)
+
+## What Data Sharing Method Works Best for My Collaboration?
+
+Choosing the right data sharing tool/method depends on how often you transfer data and who needs access to it. Below are some common scenarios and our recommendations.
+
+## Infrequent Transfer of Your Data
+
+If you are sending data Infrequently, for example, to share results with a collaborator, think about whether the transfer is a copy or a move.
+
+- If the transfer is a copy: You are keeping your own copy. Keep in mind that your version and the recipient’s version may be out of sync over time.
+
+- If the transfer is a move: After confirming the recipient has retrieved the data, it is good practice to delete your local copy to free up storage space.
+
+- **Recommended method:** [Globus](../transfer/globus.md) is the easiest and most robust option. It is also simple to restrict who has access.
+    - Put the data into a special directory.
+    - [Create a shared Globus collection](../transfer/globus.md#creating-a-guest-collection).
+    - [Sharing the collection](../transfer/tutorial/globus_organization_tutorial.md#how-do-i-share-a-collection-with-others) to the intended recipient.
```

Comment:
Because these are instruction list items, write the link text in imperative mood. In other words, tell them what to do.

> Share the collection